lib/home_work_8/task_3.dart

```diff
@@ -0,0 +1,12 @@
+import 'package:dart_learning/util.dart';
+import 'package:word_generator/word_generator.dart';
+
+void runTask3() {
+  printTaskDivider(8, 3);
+  final generator = WordGenerator();
+
+  final nounsList = generator.randomNouns(50);
+  final nounsMap = { for (var element in nounsList) element : element.length };
+  final tempNouns = {for (var element in nounsMap.entries) if(element.value%2==0) element.key : element.value};
```

Comment:
You can also try getter isEven
```suggestion
  final tempNouns = {for (var element in nounsMap.entries) if(element.isEven) element.key : element.value};
```